community/elasticsearch/APKBUILD

```diff
@@ -19,6 +20,7 @@ source="https://artifacts.elastic.co/downloads/$pkgname/$pkgname-$pkgver.tar.gz
 builddir="$srcdir/$pkgname-$pkgver"
 
 default_module=transport-netty4
+mkdir -p $builddir/modules
 _modules=$(find $builddir/modules -mindepth 1 -maxdepth 1 -type d ! -name $default_module -exec basename {} \;)
```

Comment:
does this even work? It does not look good at all.

Remember that other scripts will parse/execute the APKBUILD to figure out buildtime dependencies. I don't think we want create and directories at that point.

Then as second step the `_modules=$(find $builddir/modules .. )` will always run on an empty, newly created dir, so we could simplify with:
````diff
diff --git a/community/elasticsearch/APKBUILD b/community/elasticsearch/APKBUILD
index ecf27f1f8a..47bbb97805 100644
--- a/community/elasticsearch/APKBUILD
+++ b/community/elasticsearch/APKBUILD
@@ -20,13 +20,6 @@ source="https://artifacts.elastic.co/downloads/$pkgname/$pkgname-$pkgver.tar.gz
 builddir="$srcdir/$pkgname-$pkgver"

 default_module=transport-netty4
-mkdir -p $builddir/modules
-_modules=$(find $builddir/modules -mindepth 1 -maxdepth 1 -type d ! -name $default_module -exec basename {} \;)
-for _mod in $_modules; do
-       subpackages="$subpackages $pkgname-$_mod:_${_mod//-/_}"
-       eval "_${_mod//-/_}() { _builtin_module $_mod; }"
-done
-
 _basedir="/usr/share/java/$pkgname"

 build() {
````

and it should still give the exact same result.